src/google/protobuf/compiler/objectivec/objectivec_helpers.cc

```diff
@@ -1169,6 +1173,15 @@ bool ValidateObjCClassPrefixes(const std::vector<const FileDescriptor*>& files,
   }
 
   for (int i = 0; i < files.size(); i++) {
+    bool should_skip =
+      (std::find(generation_options.expected_prefixes_suppressions.begin(),
+                 generation_options.expected_prefixes_suppressions.end(),
+                 files[i]->name())
+          != generation_options.expected_prefixes_suppressions.end());
+    if (should_skip) {
+      return true;
```

Comment:
Should this be 'return true', or should this be a 'continue'? Don't we want to validate all of the files?

Reply:
done.